docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -170,6 +170,17 @@ do not have a dedicated frozen tier, you must configure the
 cache on one or more nodes. Partially mounted indices are only allocated to
 nodes that have a shared cache.
 
+[[manually-mounting-snapshots]]
+[WARNING]
+.Manual snapshot mounting
+====
+Manually mounting snapshots captured by an Index Lifecycle Management ({ilm-init}) policy can
+interfere with {ilm-init}'s automatic management. This may lead to issues such as data loss
+or complications with snapshot handling. For optimal results, allow {ilm-init} to manage
+snapshots automatically. If manual mounting is necessary, be aware of its potential
+impact on {ilm-init} processes. For more information, learn about <<index-lifecycle-management,{ilm-init} snapshot management>>.
+====
```

Comment:
consider breaking this into a couple of paragraphs. I'm also unclear on whether the complications you mentioned in your last draft ("can disrupt ILM actions, like deletions or phase transitions") is totally covered by this statement.

I'd avoid restating the problem, as you do in the "if you need to mount manually" sentence. it seems to imply that there are impacts not listed here. if it is meant to imply that, we should probably list the additional impacts as I don't think they're captured elsewhere.

finally, I don't think your link is ideal - there's no snapshot info on that top level ILM page. Consider a different link, or changing your link text to refer to just ILM.

```suggestion
====
Manually mounting snapshots captured by an Index Lifecycle Management ({ilm-init}) policy can
interfere with {ilm-init}'s automatic snapshot management. This may lead to issues such as data loss
or complications with snapshot handling.

For optimal results, allow {ilm-init} to manage
snapshots automatically. 

<<index-lifecycle-management,Learn more about {ilm-init} snapshot management>>.
====
```